digits/utils/image.py

```diff
@@ -306,6 +307,7 @@ def get_layer_vis_square(data,
     normalize -- whether to normalize the data when visualizing
     max_width -- maximum width for the vis_square
     """
+    assert channel_order == 'RGB' or channel_order == 'BGR'
```

Comment:
How about:
```python
if channel_order not in ['RGB', 'BGR']:
  raise ValueError('Unsupported channel_order %s' % channel_order)
```
> Raised when a built-in operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as IndexError.
https://docs.python.org/2/library/exceptions.html#exceptions.ValueError

I've got `asserts` throughout the code, but I'm trying to phase them out where appropriate.